src/python-bindings/common2/py_util.cpp

```diff
@@ -1 +1,265 @@
+PyObject_Handle *
+get_handle_from(PyObject * py) {
+	auto * py_handle = PyObject_GetAttrString( py, "_handle" );
+	Py_DecRef(py_handle);
+	return (PyObject_Handle *)py_handle;
+}
 
+
+PyObject *
+py_new_htcondor2_classad(void * classAd) {
+	static PyObject * py_htcondor2_module = NULL;
+	if( py_htcondor2_module == NULL ) {
+		 py_htcondor2_module = PyImport_ImportModule( "htcondor2" );
```

Comment:
Make "htcondor2" and "classad" manifest constants.

Reply:
Changing this in this branch turns into a merge-conflict mess, so I moved doing this to the follow-up ticket: https://opensciencegrid.atlassian.net/browse/HTCONDOR-2163